### ISSUE TITLE
Plugin causes a crash during IntelliJ indexing

### ISSUE THREAD

**Issue by mikehearn** [Darkyenus/glsl4idea#133]
See here for stack traces:

https://youtrack.jetbrains.com/issue/IDEA-186936

With this plugin activated in the latest versions of IntelliJ, the IDE tries to index .frag files inside the JDK itself (in javafx). It cannot do that and crashes.

**Comment by Darkyenus** [comment]
Hi, thanks for the report. I have already encountered those, or at least something similar to them, but since they originate from deep inside the IDE, I have no idea how to debug/fix them. Also, last time I tried to debug this, it seemed impossible, possibly a result of multiple threads accessing an object that is not thread-safe (and is not even required to be).

If someone well versed in IntelliJ's internals would be willing to help me debug this (or better, send a PR), I would be grateful.

For the time being, these crashes can be safely ignored, as they do not seem to impair any functionality and only clutter logs.
